public/app/core/internationalization/index.tsx

```diff
@@ -54,6 +55,8 @@ export function initializeI18n(language: string): Promise<{ language: string | u
     .use(initReactI18next) // passes i18n down to react-i18next
     .init(options);
 
+  tFunc = tFunc = i18n.t;
```

Comment:
What's with the multiple assignments? 🤔

Reply:
argh yikes!

I think it was a poor copy-paste job....